src/main/java/org/assertj/assertions/generator/BaseAssertionGenerator.java

```diff
@@ -345,6 +350,16 @@ private String fillAssertClassTemplate(String template, ClassDescription classDe
     return template;
   }
 
+  private String determineGeneratedSignature() {
+    if (generatedAnnotationSource == JAVAX) {
+      return "@javax.annotation.Generated(value=\"assertj-assertions-generator\")";
```

Comment:
ℹ️ Not sure when this changed and what the correct extension here is, but FYI: as of JDK 21, it's `@javax.annotation.processing.Generated`.

Reply:
This has been applied in #239.